projects/core/components/loader/loader.style.less

```diff
@@ -146,6 +146,13 @@
     overflow: hidden;
     transform: scale(1, -1);
     animation: tuiLoaderRotate 3s linear infinite;
+
+    /* Safari Only */
+    @supports (-webkit-hyphens: none) {
+        & {
+            overflow: visible;
```

Comment:
Maybe it can be used in all browsers? What's the point of overflow: hidden in chrome?

Reply:
`What's the point of overflow: hidden`

if I use `overflow: visible` in chrome

![image](https://github.com/taiga-family/taiga-ui/assets/12021443/da353b22-6d48-49f3-a059-42eb8a37ef4e)

But in Safari works as expected